sinonjs/sinon
# Upgrade jekyll dependencies

Update `jekyll` and `github-pages` in order to fix warning of CVE-2020-14001

In collaboration with @olleolleolle, I have upgraded the dependencies and verified that this works locally.

#### How to verify
1. Check out this branch
2. `bundle install`
3. `cd docs`
4. `bundle exec jekyll serve`
5. Navigate to http://127.0.0.1:4000/
6. Observe that the site still renders as normal

#### Checklist for author
